### ISSUE TITLE
Do something with dependency error reporting already

### ISSUE THREAD

**Issue by kix** [composer/composer#1649]
That's the "error explanations" I use to get:
```
  Problem 1
    - Installation request for doctrine/doctrine-bundle 1.2.* -> satisfiable by doctrine/doctrine-bundle v1.2.0-beta1.
    - Conclusion: don't install symfony/symfony v2.2.0
    - Conclusion: don't install symfony/symfony v2.2.0-RC3
    - Conclusion: don't install symfony/symfony v2.2.0-RC2
    - Installation request for phpunit/phpunit 3.7.13 -> satisfiable by phpunit/phpunit 3.7.13.
    - Conclusion: don't install symfony/symfony v2.2.0-RC1
    - Conclusion: don't install symfony/symfony v2.2.0-BETA2
    - doctrine/doctrine-bundle v1.2.0-beta1 requires symfony/framework-bundle >=2.2.0-beta2,<2.3-dev -> satisfiable by symfony/symfony v2.2.0, symfony/symfony v2.2.0-BETA2, symfony/symfony v2.2.0-RC1, symfony/symfony v2.2.0-RC2, symfony/symfony v2.2.0-RC3, symfony/framework-bundle v2.2.0, symfony/framework-bundle v2.2.0-BETA2, symfony/framework-bundle v2.2.0-RC1, symfony/framework-bundle v2.2.0-RC2, symfony/framework-bundle v2.2.0-RC3.
    - don't install symfony/framework-bundle v2.2.0|don't install symfony/symfony v2.2.0-BETA1
    - don't install symfony/framework-bundle v2.2.0-BETA2|don't install symfony/symfony v2.2.0-BETA1
    - don't install symfony/framework-bundle v2.2.0-RC1|don't install symfony/symfony v2.2.0-BETA1
    - don't install symfony/framework-bundle v2.2.0-RC2|don't install symfony/symfony v2.2.0-BETA1
    - don't install symfony/framework-bundle v2.2.0-RC3|don't install symfony/symfony v2.2.0-BETA1
    - Installation request for symfony/symfony 2.2.* -> satisfiable by symfony/symfony v2.2.0, symfony/symfony v2.2.0-BETA1, symfony/symfony v2.2.0-BETA2, symfony/symfony v2.2.0-RC1, symfony/symfony v2.2.0-RC2, symfony/symfony v2.2.0-RC3.
```
So, how do I see it? Well, obviously it states that to install ```doctrine/doctrine-bundle 1.2.*``` it can use 1.2.0 version. I don't really see how that's an error or anything, OK, I'll assume it's just info. But why the hell does it decide not to install symfony 2.2.0 then? That's the requirement I obviously stated in my ```composer.json```, yet Composer ignores my request.

And what for do I need all these bollocks on all the possible existing beta versions and whatnot? 

Do something with these messages. They should read like they're actually aimed at human understanding, but now they're just cryptic.

**Comment by hkdobrev** [comment]
This was probably fixed in #5120.

**Comment by Seldaek** [comment]
Yup.
